includes/Abilities/Image/Import_Base64_Image.php

```diff
@@ -0,0 +1,281 @@
+<?php
+/**
+ * Base64 encoded image import WordPress Ability implementation.
+ *
+ * @package WordPress\AI
+ */
+
+declare( strict_types=1 );
+
+namespace WordPress\AI\Abilities\Image;
+
+use Throwable;
+use WP_Error;
+use WordPress\AI\Abstracts\Abstract_Ability;
+use WordPress\AiClient\Files\DTO\File;
+
+/**
+ * Base64 encoded image import WordPress Ability.
+ *
+ * @since x.x.x
+ */
+class Import_Base64_Image extends Abstract_Ability {
+
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @since x.x.x
+	 */
+	protected function input_schema(): array {
+		return array(
+			'type'       => 'object',
+			'properties' => array(
+				'data'        => array(
+					'type'              => 'string',
+					'sanitize_callback' => 'sanitize_text_field',
+					'description'       => esc_html__( 'The base64 encoded image data to import into the media library.', 'ai' ),
+				),
+				'filename'    => array(
+					'type'              => 'string',
+					'sanitize_callback' => 'sanitize_text_field',
+					'description'       => esc_html__( 'The filename of the image.', 'ai' ),
+				),
+				'title'       => array(
+					'type'              => 'string',
+					'sanitize_callback' => 'sanitize_text_field',
+					'description'       => esc_html__( 'The title of the image.', 'ai' ),
+				),
+				'description' => array(
+					'type'              => 'string',
+					'sanitize_callback' => 'sanitize_text_field',
+					'description'       => esc_html__( 'The description of the image.', 'ai' ),
+				),
```

Comment:
In addition to these, we should also encourage (at least support) passing an alt text, crucial for accessibility. It can be set via `_wp_attachment_image_alt` below.